src/main/java/fm/last/moji/impl/MojiImpl.java

```diff
@@ -72,7 +72,7 @@ public void copyToMogile(File source, MojiFile destination) throws IOException {
       outputStream.flush();
     } finally {
       IOUtils.closeQuietly(inputStream);
-      IOUtils.closeQuietly(outputStream);
+      outputStream.close();;
```

Comment:
I will post the full stacktrace on monday, when I'm at work again.

> you're saying that the exception you mention in the other PR is being swallowed here
> So if we rethrow it instead of swallowing then you should have enough information

Actually I thought so as well. However at somepoint the message changes from `Content length required` to `size_verifiy_error`, which is still puzzling me.